src/chemcoord/_cartesian_coordinates/xyz_functions.py

```diff
@@ -84,6 +85,104 @@ def open_file(i: int) -> None:
         raise ValueError("Argument is neither list nor Cartesian.")
 
 
+@overload
+def multiple_to_xyz(
+    cartesian_list: Sequence[Cartesian],
+    buf: None = None,
+    sort_index: bool = ...,
+    overwrite: bool = ...,
+    float_format: Callable[[float], str] = ...,
+) -> str: ...
+
+
+@overload
+def multiple_to_xyz(
+    cartesian_list: Sequence[Cartesian],
+    buf: PathLike,
+    sort_index: bool = ...,
+    overwrite: bool = ...,
+    float_format: Callable[[float], str] = ...,
+) -> None: ...
+
+
+def multiple_to_xyz(
+    cartesian_list: Sequence[Cartesian],
+    buf: Union[PathLike, None] = None,
+    sort_index: bool = True,
+    overwrite: bool = True,
+    float_format: Callable[[float], str] = "{:.6f}".format,
+) -> Union[str, None]:
+    """Write a list of Cartesians into an xyz file.
+
+    .. note:: Since it permamently writes a file, this function
+        is strictly speaking **not sideeffect free**.
+        The list to be written is of course not changed.
+
+    Args:
+        cartesian_list (list):
```

Comment:
After #144 is merged, sphinx will take the type information from the type hinting, i.e. you can just write

```
cartesian_list :
```
instead of 
```
cartesian_list (list) :
```

Since some of the types are wrong now, I would just remove them and let the type hints do their job.